src/components/compounds/ArticleContainer/index.js

```diff
@@ -42,8 +44,14 @@ class ArticleContainer extends Component {
     return null;
   }
 
+  handlePageClick = (pageNumber) => {
```

Comment:
### Nitpick

You should keep functions like this outside the render method.

Reply:
Duly noted.